developer_manual/digging_deeper/index.rst

```diff
@@ -0,0 +1,24 @@
+==============
+Digging deeper
+==============
+
+.. toctree::
+   :maxdepth: 2
+
+   api
+   changelog
+   debugging
+   classloader
+   flow
+   javascript-apis
+   npm
+   notifications
```

Comment:
Maybe we should move those kind of integrations like notifications, flow, 2fa to a separate sub-category?

Reply:
Just for integrations? I guess that would make sense as well. My consideration was that *Digging deeper* contains the specific stuff you don't need for the average app, but will look up if really needed. Integrations are part of this in a way, I think.

Reply:
I was more thinking of even lowering the number of sections. Like combining *Getting started* and *The basics* :thinking:

Reply:
So I'm really on the fence with this :speak_no_evil: 

Let's think about it. We can move things around later on.